string object_name
---
float32 grip_force